source/print.cpp

```diff
@@ -27,6 +27,8 @@
 #include "print.h"
 
 #if defined(SPIRV_LINUX) || defined(SPIRV_MAC)
+namespace libspirv {
```

Comment:
I think this can be moved above the `#if` macro check so we don't need to have to duplicate it again for the windows case.